code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -80,6 +80,12 @@
 	caliber = "cap"
 	max_ammo = 7
 
+/obj/item/ammo_box/magazine/internal/overgrown
+	name = "overgrown pistol magazine"
+	desc= "Oh god, this shouldn't be here"
```

Comment:
```suggestion
	des c= "Oh god, this shouldn't be here."
```

Reply:
I copy pasted this from the rest of the descriptions, I added the missing space, but didn't do the "." as it's not intended to be seen by anyone. If you really want the punctuation, I'll also go add it to the rest of the internal mags.